OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -542,6 +542,15 @@ void trackValuePropLearnMoreTapped(@NonNull String courseId, @Nullable String as
      */
     void trackResumeCourseBannerTapped(@NonNull String courseId, @NonNull String blockId);
 
+    /**
+     * Track the View on Web tapped on Subsection screen in case of non supported xBlocks
+     *
+     * @param courseId          Id of the course
+     * @param blockId           Id of the non-supported xBlock.
+     * @param isSpecialExamInfo Whether the block is special exam info ot not
```

Comment:
typo `@param isSpecialExamInfo Whether the subsection is special exam info or not`